diff --git a/ui/src/core/embedder/default_plugins.ts b/ui/src/core/embedder/default_plugins.ts
index 46bb22e..0306dfa 100644
--- a/ui/src/core/embedder/default_plugins.ts
+++ b/ui/src/core/embedder/default_plugins.ts
@@ -98,6 +98,7 @@ export const defaultPlugins = [
   'dev.perfetto.TraceProcessorTrack',
   'dev.perfetto.TrackEvent',
   'dev.perfetto.TrackUtils',
+  'io.company.RELUPerfettoBridge',
   'org.Chromium.OpenTableCommands',
   'org.kernel.LinuxKernelSubsystems',
   'org.kernel.SuspendResumeLatency',
